infoschema/tables_test.go

```diff
@@ -956,6 +956,22 @@ func (s *testTableSuite) TestFormatVersion(c *C) {
 		version := infoschema.FormatVersion(v, false)
 		c.Assert(version, Equals, res[i])
 	}
+
+	versions = []string{"v4.0.12", "4.0.12", "v5.0.1"}
+	resultVersion := []string{"4.0.12", "4.0.12", "5.0.1"}
+
+	f := func(v string) string {
+		if len(v) >= 1 && v[0] == 'v' {
+			v = v[1:]
+		}
+
+		return v
+	}
```

Comment:
Well done!
But I think it's not a good idea to copy the code block. Maybe you can create a function like `FormatTiFlashVersion`, just like what `FormatVersion` does?
Besides, I think `FormatVersion` needs to be renamed to `FormatTiDBVersion`.

Reply:
Indeed, it's more elegant, thanks for your suggestion